Git/podspec-merge-driver.py

```diff
@@ -0,0 +1,52 @@
+#!/usr/bin/env python
+
```

Comment:
Please add a bit of documentation to explain what those scripts are for and what they do. So that a random person finding the scripts can know what it is. We can imagine doing that in a README since both files here are related to each other.

Reply:
I was thinking a readme would be helpful, will add it